autogen/retrieve_utils.py

```diff
@@ -297,3 +338,71 @@ def query_vector_db(
         where_document={"$contains": search_string} if search_string else None,  # optional filter
     )
     return results
+
+def create_lancedb_from_dir(dir_path, max_tokens, db_path, table_name, chunk_mode,
```

Comment:
duplicate code for `create_lancedb_from_dir` and `query_lancedb`.

Reply:
yes removed it.my bad